Gems/ROS2/Code/Source/ROS2EditorModule.cpp

```diff
@@ -11,6 +11,12 @@
 #include <Camera/ROS2CameraSensorEditorComponent.h>
 #include <RobotImporter/ROS2RobotImporterEditorSystemComponent.h>
 
+void InitROS2Resources()
+{
+    // Registration of Qt (ROS2.qrc) resources
+    Q_INIT_RESOURCE(ROS2);
```

Comment:
Where is this macro defined? I am wondering whether we can depend on its visibility based on included headers.

Reply:
This macro is defined in QtCore/qglobal.h. I can include it in this file to make sure it's always defined.